### PR TITLE
Set true if localDiskFlag is not set

### PR DESCRIPTION
**Pull request by mattcui**
@maximilien "String" solution can't work. I got another idea, very simple, please review the code, I will add unit test if you agree with this solution. Thanks.

### CODE REVIEW

File: api/dispatcher/json.go [hunk 1]
```diff
@@ -69,6 +72,12 @@ func (c JSON) Dispatch(reqBytes []byte) []byte {
 
 	c.logger.DebugWithDetails(jsonLogTag, "Deserialized request", req)
 
+	if strings.Contains(strings.ToUpper(fmt.Sprintf("%s", req)), strings.ToUpper("localDiskFlag")) {
```
Comment (maximilien) [inline api/dispatcher/json.go]: Move this into it's own private method and add comment (in code) to why we are doing that. Along with a comment note saying that in time we will remove.

Also, add a Unit Test :)

**Comment by maximilien** [comment]
Cool @mattcui , I like this better. Please address my small comments and I will merge. Best.

**Comment by mattcui** [comment]
@maximilien Already addressed your comments, please review. Thanks.

File: api/dispatcher/json.go [hunk 2]
```diff
@@ -72,11 +72,8 @@ func (c JSON) Dispatch(reqBytes []byte) []byte {
 
 	c.logger.DebugWithDetails(jsonLogTag, "Deserialized request", req)
 
-	if strings.Contains(strings.ToUpper(fmt.Sprintf("%s", req)), strings.ToUpper("localDiskFlag")) {
-		bslcommon.LocalDiskFlagNotSet = false
-	} else {
-		bslcommon.LocalDiskFlagNotSet = true
-	}
+	// Will remove this line after
```
Comment (maximilien) [inline api/dispatcher/json.go]: explain please

Comment (mattcui) [inline api/dispatcher/json.go]: Sorry, it's an unfinished comment, already corrected it, please review. Thanks.

**Pull request merged**